exercises/all-your-base/example.rs

```diff
@@ -55,6 +55,11 @@ pub fn convert<P: AsRef<[Digit]>>(
     if let Some(&invalid) = digits.as_ref().iter().find(|&num| *num >= from_base) {
         return Err(Error::InvalidDigit(invalid));
     }
+    
+    // no output digits correspond to zero
+    if digits.as_ref().is_empty() {
+        return Ok(vec![0])
+    }
```

Comment:
This check is redundant. I understand that it is slightly faster in the case that the digits list is empty, but for the examples, I think DRY is more important than optimization.

Reply:
I've removed this check here: 056f7c2.